.github/ISSUE_TEMPLATE/service-proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Service proposal
+about: Template to use to submit a service proposal
+title: "[SERVICE]"
+labels: service
+assignees: ''
+
+---
+
+<!-- 
+
+Use the following template when submitting proposals for new CUSTOM services to this repository.
```

Comment:
so are they custom or can they also be standard services?

Reply:
They can be standard, but most of the details can be excluded since the official specification document can be linked to instead. 

The more explicit template is for custom services that do not already have a specification document available.

Perhaps we can say: "Use this template to provide details of the proposed service if there is not an existing specification document available."